jOOQ/src/main/java/org/jooq/Clause.java

```diff
@@ -48,6 +48,8 @@
 
     USER,
     ROLE,
+    PRIVILEGE,
+    GRANT,
```

Comment:
The `GRANT` clause will need to be reviewed. `GRANT` should be the statement prefix, and then there will be clauses inside of it. Also it will need documentation. I'll take care of this after merging.

Reply:
Yes